UserAgent/Home View/HistoryView.swift

```diff
@@ -135,7 +135,7 @@ private extension HistoryView {
         emptyLabel.snp.makeConstraints { make in
             make.centerX.equalTo(overlayView)
             // Sets proper top constraint for iPhone 6 in portait and for iPad.
-            make.centerY.equalTo(overlayView).offset(LibraryPanelUX.EmptyTabContentOffset).priority(100)
+            make.centerY.equalTo(overlayView).offset(-180).priority(100)
```

Comment:
Please move 
struct LibraryPanelUX {
    static let EmptyTabContentOffset = -180
} 
structure in LibraryView.swift file and rename LibraryViewUX as downloads view is also using this value it will be better to have property for this.